website/content/blog/announcing-the-platform-engineering-maturity-model.md

```diff
@@ -0,0 +1,82 @@
+---
+title: 'Announcing the Platform Engineering Maturity Model'
+date:   2023-11-01 00:00:00 +0000
+author: Abby Bangser, Josh Gavant
+---
+
+The CNCF Platforms Working Group (WG) is excited to present the first release of a platform engineering maturity model which provides a more concrete application of the extremely well received white paper from this past April.
```

Comment:
Perhaps add a hyperlink here on white paper to <https://tag-app-delivery.cncf.io/whitepapers/platforms/> rather than (or in addition to) the next mention.